Sources/ArcGISToolkit/Components/AR/WorldScaleSceneView.swift

```diff
@@ -0,0 +1,207 @@
+                    )
+                    sceneViewProxy.setFieldOfView(
+                        for: frame,
+                        orientation: interfaceOrientation
+                    )
+                    
+                    sceneViewProxy.draw()
+                }
+            
+            if initialCameraIsSet {
+                SceneViewReader { proxy in
+                    sceneViewBuilder(proxy)
+                        .cameraController(cameraController)
+                        .attributionBarHidden(true)
+                        .spaceEffect(.transparent)
+                        .atmosphereEffect(.off)
+                        .viewDrawingMode(.manual)
+                        .onCameraChanged { camera in
+                            self.currentCamera = camera
+                        }
+                        .onAppear {
+                            // Capture scene view proxy as a workaround for a bug where
+                            // preferences set for `ARSwiftUIView` are not honored. The
+                            // issue has been logged with a bug report with ID FB13188508.
+                            self.sceneViewProxy = proxy
+                        }
+                }
+            }
+        }
+        .overlay(alignment: .top) {
+            if !statusText.isEmpty {
+                Text(statusText)
+                    .multilineTextAlignment(.center)
+                    .frame(maxWidth: .infinity, alignment: .center)
+                    .padding(8)
+                    .background(.regularMaterial, ignoresSafeAreaEdges: .horizontal)
+            }
+        }
+        .observingInterfaceOrientation($interfaceOrientation)
+        .onAppear {
+            arViewProxy.session.run(configuration)
+        }
+        .onDisappear {
+            arViewProxy.session.pause()
+        }
+        .task {
+            do {
+                withAnimation {
+                    statusText = "Acquiring current location."
+                }
+                try await locationDatasSource.start()
+                await withTaskGroup(of: Void.self) { group in
+                    group.addTask {
+                        for await location in locationDatasSource.locations {
+                            await updateSceneView(for: location)
+                        }
+                    }
+                }
+            } catch {
+                withAnimation {
+                    statusText = "Failed to acquire current location."
+                }
+            }
+        }
+    }
+    
+    /// If necessary, updates the scene view's camera controller for a new location coming
+    /// from the location datasource.
+    @MainActor
+    private func updateSceneView(for location: Location) {
+        // Make sure there is at least a minimum horizontal and vertical accuracy.
+        guard location.horizontalAccuracy < 10 && location.verticalAccuracy < 10 else { return }
```

Comment:
The scene content is not rendering since my horizontal accuracy is 35 meters and vertical accuracy is ~16 meters outdoors. Are these values too low or is there a way to calibrate the device compass?

Reply:
It would be unrelated to the compass, it's the GPS receiver. You can try again in a little while and see if some satellites come into view.